Bunkum.HttpServer/BunkumHttpServer.cs

```diff
@@ -97,12 +79,9 @@ private void RunStartupTasks()
         stopwatch.Start();
         
         this._logger.LogInfo(BunkumContext.Startup, "Starting up...");
-        if (this._authenticationProvider is DummyAuthenticationProvider && this.AssumeAuthenticationRequired)
-        {
-            this._logger.LogWarning(BunkumContext.Startup, "The server was started with a dummy authentication provider. " +
-                                                            "If your application relies on authentication, users will always have full access.");
-        }
         
+        foreach (Service service in this._services) service.Initialize();
```

Comment:
Ideally we would log here, like we do right below for initializing the database provider